src/kaltura-player.js

```diff
@@ -121,6 +121,7 @@ class KalturaPlayer extends FakeEventTarget {
    * kalturaPlayer.loadMedia({entryId: 'entry123'}, {startTime: 5, poster: 'my/poster/url'});
    */
   loadMedia(mediaInfo: ProviderMediaInfoObject, mediaOptions?: PKSourcesConfigObject): Promise<*> {
+    this._reset = false;
```

Comment:
pretty risky. `reset` called a few lines below so it gonna change the loadMedia logic.
can we do it in critical error only?